### DISCUSSION TITLE
Is possible add more than1 banner or slider on homepag?

### DISCUSSION THREAD

**Discussion by minhcz** [PrestaShop/PrestaShop#34108] [Q&A] [answered]
Hello,

I am new with prestashop, my question is if possible add more than 1 banner or text block on homepage with default module from prestashop dev? or i need to use other module support to do that?

Thank you.

**Comment by SharakPL** [comment, marked as answer]
You can use the same slider or banner in many places, but AFAIK you can't do different ones with same modules. You would have to create a copy, rename it along with some variables and install it as separate module.

You could also use one of available modules for this, eg. Creative Elements.
